packages/components/ScreenContainer/Header.tsx

```diff
@@ -98,9 +123,38 @@ export const Header: React.FC<{
             {children}
           </>
         )}
+        <View
+          style={{
+            flex: 1,
+            flexDirection: "row",
+            height: headerHeight,
+            alignItems: "center",
+          }}
+        >
+          <TogglePlayerButton />
+          {media && (
+            <Separator
+              horizontal
+              style={{ height: "100%", marginHorizontal: layout.spacing_x2 }}
+            />
+          )}
+          <SearchBar style={{ marginLeft: media ? 0 : 60, flex: 1 }} />
```

Comment:
Because @zxxma listed this change in the issue, i create a specific commit for it in this PR (even if you said that you didn't do atomic commits anymore, and more atomic PR's, i found that appropriate anyway), so i don't have opinion on that actually 👍 as you want :)

Reply:
please extract it then, it's breaking some screens headers currently and it's something we can isolate and do properly

Reply:
Okay no problem 👍